cms/templates/partials/admissions-section.html

```diff
@@ -0,0 +1,43 @@
+{% load image_version_url wagtailcore_tags %}
+<div class="container-fluid mb-4 admissions-section">
+    <div class="row">
+        <div class="container admissions-section">
+            <div class="row">
+                <div class="col-md-5 col-sm-12 mb-2">
+                    <h1 class="section-heading">Information</h1>
+                    <h2 class="admissions-heading mt-2">Admissions</h2>
```

Comment:
I guess, it should link to `/admissions `

Reply:
Will do.